report/renderer/performance-category-renderer.js

```diff
@@ -312,18 +312,18 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       ...filterableMetrics,
     ]);
     for (const metric of filterChoices) {
-      const elemId = `metric-${metric.acronym}`;
```

Comment:
another option is to keep the previous id / htmlFor stuff but tack on a unique identifier here. 

```js
const elemId = `metric-${metric.acronym}-${Math.random()}`;
const elemId = `metric-${metric.acronym}-${Math.random()-${Date.now()}`;
```

or somehow wire data from the LHR here.